src/js/components/Banner.jsx

```diff
@@ -0,0 +1,26 @@
+import {bannerText} from '../../../config'
+import {grey800, yellow200} from 'material-ui/styles/colors'
+import React, {Component} from 'react'
+
+const style = {
+  banner: {
+    background: grey800,
+    color: yellow200,
+    fontSize: '0.8rem',
+    fontWeight: 400,
+    textAlign: 'center'
+  }
+}
+
+export class Banner extends Component {
```

Comment:
Google to the rescue, now that I know that there's a difference between the two: [Oh](https://medium.com/@housecor/react-stateless-functional-components-nine-wins-you-might-have-overlooked-997b0d933dbc#.t1dz2gfu2)